controller/controller.ts

```diff
@@ -1,17 +1,16 @@
-import axios from 'axios';
-import asyncRetry from 'async-retry';
-import { User } from './models';
-import { transformUser } from './utils';
+import { transformUser } from '../utils/utils';
+import { getUsers } from '../services/services';
+import { User } from '../models/models';
 
-const getUsers = async (): Promise<User[]> => {
+const getUsersController = async (): Promise<User[]> => {
```

Comment:
A controller should be either a class with methods or a constant with various functions. This is just one single function.